blank-typescript/template/App.tsx

```diff
@@ -0,0 +1,146 @@
+import React, { useCallback } from 'react';
+import {
+    SafeAreaView,
+    ScrollView,
+    StyleSheet,
+    Linking,
+    TextStyle,
+    ViewStyle,
+    ImageStyle,
+    View,
+    Animated,
+    Easing,
+} from 'react-native';
+import { Button, Divider, Provider as ThemeProvider, useTheme } from 'react-native-paper';
+import * as PXBThemes from '@pxblue/react-native-themes';
+import { Body1, Header, H4 } from '@pxblue/react-native-components';
+import { Theme } from 'react-native-paper/lib/typescript/types';
+import Logo from './assets/images/Logo.svg';
+
+const styles = (
+    theme: Theme
+): StyleSheet.NamedStyles<{
+    content: ViewStyle;
+    pxbLogoWrapper: ViewStyle;
+    pxbLogo: Animated.WithAnimatedValue<ImageStyle>; // @TODO: Verify this type
```

Comment:
`pxbLogo: ViewStyle` should be fine here — the styles are being put onto a View component (the fact that it's animated shouldn't affect the available style props).